scripts/devnet-curio/.env

```diff
@@ -0,0 +1,24 @@
+CURIO_IMAGE=curio/curio-dev:dev
```

Comment:
It would be safer to pin it to a concrete hash.

Reply:
Or is it a local docker build?

Reply:
It's local. They do not upload their docker images. It's written in the README.md how to build the `curio/curio-dev` image.

Reply:
Would it make sense to make _our_ build of curio an upload it, along the lines of https://github.com/ChainSafe/forest/blob/main/.github/workflows/lotus-devnet-publish.yml?

This would allow embedding it in the CI. Otherwise, it's rather not reproducible.

Reply:
Yes, this needs to be done eventually.